main/src/ca/uwaterloo/flix/language/phase/Resolver.scala

```diff
@@ -848,6 +848,18 @@ object Resolver extends Phase[NamedAst.Root, ResolvedAst.Program] {
           for {
             es <- traverse(elms)(visit)
           } yield ResolvedAst.Pattern.Array(es, tvar, loc)
+
+        case NamedAst.Pattern.ArrayTailSpread(elms, sym, tvar, loc) =>
+          val array = for {
+            es <- traverse(elms)(visit)
+          } yield ResolvedAst.Pattern.Array(es, tvar, loc)
+          ResolvedAst.Pattern.ArrayTailSpread(array.get.elms, sym, array.get.tvar, loc).toSuccess
+
+        case NamedAst.Pattern.ArrayHeadSpread(sym, elms, tvar, loc) =>
+          val array = for {
+            es <- traverse(elms)(visit)
+          } yield ResolvedAst.Pattern.Array(es, tvar, loc)
+          ResolvedAst.Pattern.ArrayHeadSpread(sym, array.get.elms, array.get.tvar, loc).toSuccess
```

Comment:
This is incorrect. You should have this part inside the yield expression. You are not allowed to call get this way.